Vizzuality/airqast
# Demos: disable scroll-to-zoom on the maps

This PR disables the ability to scroll to zoom on the maps of the demos.

In addition, this PR fixes a small error that hasn't caused any issue yet: the options of the map would always be overwritten by the defaults.

## Testing instructions
1. Go to http://localhost:3000/products/emissions/demo
2. Once the demo has loaded, place your cursor at the center of the screen and scroll to the bottom of the page

The map shouldn't intercept the scroll event and you should be able to scroll normaly. Please also test the other demo.

## Pivotal

[Task](https://www.pivotaltracker.com/story/show/160444570)
